libnetwork/drivers/bridge/bridge_linux.go

```diff
@@ -513,7 +513,7 @@ func (d *driver) configure(option map[string]interface{}) error {
 		removeIPChains(iptables.IPv4)
 
 		if err := setupHashNetIpset(ipsetExtBridges4, unix.AF_INET); err != nil {
-			return err
+			return fmt.Errorf("creating ipset "+ipsetExtBridges4+": %w (kernel modules ip_set, ip_set_hash_net and netfilter_xt_set are required)", err)
```

Comment:
As we're already requiring formatting, it's probably better in this case to use `ipsetExtBridges4` as a argument, and put a `%s` here;

```suggestion
			return fmt.Errorf("creating ipset %s: %w (kernel modules ip_set, ip_set_hash_net and netfilter_xt_set are required)", ipsetExtBridges4, err)
```

Reply:
Alternatively, we could possibly include that info in the error returned by `setupHashNetIpset`, which would also allow differentiating "create" or "flush" to fail 🤔 ;  https://github.com/moby/moby/blob/a2a6bf51fae014453a8f60058270a3a052e0e529/libnetwork/drivers/bridge/bridge_linux.go#L583-L594


```go
func setupHashNetIpset(name string, family uint8) error {
	if err := netlink.IpsetCreate(name, "hash:net", netlink.IpsetCreateOptions{
		Replace: true,
		Family:  family,
	}); err != nil {
		return fmt.Errorf("creating ipset %s: %w", name, err)
	}
	if err := netlink.IpsetFlush(name); err != nil {
		return fmt.Errorf("flushing ipset %s: %w", name, err)
	}
	return nil
}
```

Reply:
Thank you - done.